src/services/calendar/index.js

```diff
@@ -1,12 +1,16 @@
 const request = require('request-promise-native');
 
 const hooks = require('./hooks');
-const { REQUEST_TIMEOUT } = require('../../../config/globals');
+const { Configuration } = require('@schul-cloud/commons');
 
 function toQueryString(paramsObject) {
-	return Object
-		.keys(paramsObject)
-		.map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(paramsObject[key])}`)
+	return Object.keys(paramsObject)
+		.map(
+			(key) =>
+				`${encodeURIComponent(key)}=${encodeURIComponent(
```

Comment:
Codacy found an issue: [Expected no linebreak before this expression.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)